### DISCUSSION TITLE
A few questions about paid subscriptions

### DISCUSSION THREAD

**Discussion by christophberger** [buttondown/discussions#33] [Q&A] [answered]
I have a few questions about paid subscription features that I cannot seem to derive from the docs.

1. If I start with a pay-as-you-wish model, can I switch to fixed tiers later?
2. Can I change the price of a paid tier if there are already subscribers on this tier?
3. If I change or add tiers, can I make obsolete tiers inaccessible but have the current subscribers grandfathered-in on the obsolete tiers? (Thinking of beta users that pay $2, and when the beta is over, the $2 tier shall disappear, but the beta users shall not have to pay more.)

**Comment by jmduke** [comment, marked as answer]
All great questions!

1. Yes. In such an event, the subscribers who were on PWYW will stick with their specific amount but all new subscribers will be on the set tiers.
2. Nope, unfortunately. This is more of a Stripe limitation than a Buttondown one; we can help you manually migrate people to the new price point, but it won't be automatic.
3. Yup!

**Comment by christophberger** [comment]
And great answers, thank you, @jmduke! 1 and 3 are what I hoped for. I can live with #2, it's just important for me to know my options.
